cli/code_scanner.py

```diff
@@ -210,13 +209,35 @@ def validate_zip_file_size(scan_type, zip_file_size):
 
 def perform_scan(cycode_client, zipped_documents: InMemoryZip, scan_type: str, scan_id: UUID, is_git_diff: bool,
                  is_commit_range: bool, scan_parameters: dict):
+    if scan_type == SCA_SCAN_TYPE:
+        return perform_scan_async(cycode_client, zipped_documents, scan_type, scan_parameters)
+
     scan_result = cycode_client.commit_range_zipped_file_scan(scan_type, zipped_documents, scan_id) \
-        if is_commit_range else cycode_client.zipped_file_scan(scan_type, zipped_documents, scan_id, scan_parameters,
-                                                               is_git_diff)
+        if is_commit_range else cycode_client.zipped_file_scan(scan_type, zipped_documents, scan_id,
+                                                               scan_parameters, is_git_diff)
 
     return scan_result
 
 
+def perform_scan_async(cycode_client, zipped_documents: InMemoryZip, scan_type: str,
+                       scan_parameters: dict) -> ZippedFileScanResult:
+    scan_async_result = cycode_client.zipped_file_scan_async(zipped_documents, scan_type, scan_parameters)
```

Comment:
whats happening if the user do SCA scan by path? you don't have the repository details. what will happen?

Reply:
no report will be produced for him and the report_url will be empty